src/Effects.js

```diff
@@ -1254,6 +1255,37 @@ module.exports = class Effects {
       }
     };
 
+    this.exploding_stars = {
+      stars: [],
+      initStars: function () {
+        for (let i = 0; i < 100; i++) {
+          let theta = p5.random(0, p5.TWO_PI);
+          let velocity = p5.random(4,12);
+          this.stars.push({
+            color: randomColor(255, 255, 100),
+            x: 200,
+            y: 200,
+            dx: velocity * p5.cos(theta),
+            dy: velocity * p5.sin(theta)
+          });
+        }
+      },
+      draw: function () {
+        p5.angleMode(p5.RADIANS);
+        p5.noStroke();
```

Comment:
What does this `noStroke` do?

Reply:
Bump

Reply:
Weird, I thought I responded to this, but must not have...
`noStroke()` prevents an outline, like 
![image](https://user-images.githubusercontent.com/8787187/67134430-fc496d80-f1c6-11e9-9df0-e5b496ed667e.png)